### PR TITLE
CPDBDS-331 | making policies optional

### PR DESCRIPTION
**Pull request by pranshu-m**
Removing dnsPolicy and restartPolicy as these properties are added by default

### CODE REVIEW

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-diskover-pr-17**

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-diskover-pr-17**

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-diskover-pr-17**

**Review by rayscott007 — APPROVED** [review]

File: charts/diskover/templates/deployment-server.yaml [hunk 1]
```diff
@@ -44,8 +44,10 @@ spec:
         #   name: localtime
         # - mountPath: /etc/timezone
         #   name: timezone
+{{- if .Values.policies.enabled }}
```
Comment (abhijit-kottur) [inline charts/diskover/templates/deployment-server.yaml]: Did you check if `policies.enabled: true` will work with latest k8s version?

File: charts/diskover/templates/deployment-worker.yaml [hunk 2]
```diff
@@ -45,8 +45,10 @@ spec:
         #   name: localtime
         # - mountPath: /etc/timezone
         #   name: timezone
+{{- if .Values.policies.enabled }}
```
Comment (abhijit-kottur) [inline charts/diskover/templates/deployment-worker.yaml]: Same here

File: charts/diskover/templates/deployment-server.yaml (hunk 1 above)
Comment (pranshu-m) [reply charts/diskover/templates/deployment-server.yaml]: on latest k8s version, enabled will be false. I have kept true here for backward compatibility.

Comment (abhijit-kottur) [reply charts/diskover/templates/deployment-server.yaml]: That would work. but what if we wanna have `dnsPolicy` and `restartPolicy` enabled on the new k8s?

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-diskover-pr-17**

File: charts/diskover/templates/deployment-server.yaml (hunk 1 above)
Comment (pranshu-m) [reply charts/diskover/templates/deployment-server.yaml]: Removed the policies, they should be included by default

**Review by abhijit-kottur — APPROVED** [review]

**Review by rayscott007 — APPROVED** [review]

**Pull request merged**